composer.json

```diff
@@ -10,7 +10,8 @@
 	"require": {
 		"composer/installers": "1.6.0",
 		"ext-openssl": "*",
-		"automattic/jetpack-logo": ">=1.0"
+		"automattic/jetpack-logo": "dev-try/custom-autoloader",
+		"automattic/autoloader": "dev-try/custom-autoloader"
```

Comment:
Most of what this package does we need it just when building the plugin, I would try to move as much as we can to `require-dev`

Reply:
great idea!

Reply:
done